.dockerignore

```diff
@@ -0,0 +1,3 @@
+Dockerfile
+docker-compose.yml
+.dockerignore

```

Comment:
You'll want to include `node_modules` in here

Reply:
yeah i was trying to avoid node_modules since it can be big but i can see why you would want to do this especially with slow internet or even none

Reply:
Oh that's a good point

Reply:
right, didn't even know docker could use mounting on top of RUN command, interesting